internal/pkg/service/stream/definition/repository/branch/branch_delete.go

```diff
@@ -0,0 +1,28 @@
+package branch
+
+import (
+	"context"
+	"time"
+
+	"github.com/keboola/go-utils/pkg/deepcopy"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/op"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
+)
+
+func (r *Repository) SoftDelete(k key.BranchKey, now time.Time, by definition.By) *op.AtomicOp[definition.Branch] {
```

Comment:
Why the module is `branch_delete` which contains `SoftDelete` when `branch_undelete` contains `Undelete` method?

Reply:
With the name `module` do you mean `file`?

Because I want to emphasize with the name of the method that it is a soft delete.